### ISSUE TITLE
Unit tests are failing

### ISSUE THREAD

**Issue by kingledion** [pastjean/postmark-rs#9]
I tried this code:

```rust
cargo test
```

I expected to see this happen: Unit tests to pass

Instead, this happened: Multiple unit tests failed trying to flatten a string Enum.

### Meta

`rustc --version --verbose`:
```
rustc 1.68.2 (9eb3afe9e 2023-03-27)
binary: rustc
commit-hash: 9eb3afe9ebe9c7d2b84b71002d44f4a0edac95e0
commit-date: 2023-03-27
host: x86_64-unknown-linux-gnu
release: 1.68.2
LLVM version: 15.0.6
```

**Comment by kingledion** [comment]
https://github.com/pastjean/postmark-rs/pull/10

**Comment by pastjean** [comment]
Fixed by #10
